src/main/java/frc/robot/subsystems/BlingSubsystem.java

```diff
@@ -12,7 +12,9 @@
 import com.ctre.phoenix.led.CANdle;
 import com.ctre.phoenix.led.CANdleConfiguration;
 import com.ctre.phoenix.led.RainbowAnimation;
+import com.ctre.phoenix.led.RgbFadeAnimation;
```

Comment:
I don't think you still need to import these Animations, since they are in SetBlingCommand class now.
So remove them from BlingSubsystem.